src/main/java/it/pagopa/interop/probing/eservice/operations/util/EnumUtilities.java

```diff
@@ -11,38 +11,34 @@
 import it.pagopa.interop.probing.eservice.operations.dtos.EserviceInteropState;
 import it.pagopa.interop.probing.eservice.operations.dtos.EserviceMonitorState;
 import it.pagopa.interop.probing.eservice.operations.model.view.EserviceView;
+import it.pagopa.interop.probing.eservice.operations.util.constant.ProjectConstants;
 
 @Component
 public class EnumUtilities {
 
-  @Value("${minutes.ofTollerance.multiplier}")
-  private int minOfTolleranceMultiplier;
+  @Value("${toleranceMultiplierInMinutes}")
+  private int toleranceMultiplierInMinutes;
 
   public static String fromMonitorToPdndState(EserviceMonitorState state) {
-    switch (state) {
-      case ONLINE:
-        return EserviceInteropState.ACTIVE.getValue();
-      case OFFLINE:
-        return EserviceInteropState.INACTIVE.getValue();
-      default:
-        return null;
-    }
+    return switch (state) {
+      case ONLINE -> EserviceInteropState.ACTIVE.getValue();
+      case OFFLINE -> EserviceInteropState.INACTIVE.getValue();
+      case N_D -> ProjectConstants.N_D;
+      default -> throw new IllegalArgumentException("Invalid state {}= " + state);
+    };
   }
 
   public List<String> convertListFromMonitorToPdnd(List<EserviceMonitorState> statusList) {
```

Comment:
how about to use Collection instead of List as a result for functions?

Reply:
It's not necessary since the lists are of the right type for the use cases. If the requirement will change, then I will make the change